lib/package/config.coffee

```diff
@@ -51,6 +51,11 @@ config =
     description: "Limits the Console history's size."
     default: 10000
     order: 8
+  cellDelimiter:
+    type: 'string'
+    default: '#---'
```

Comment:
Maybe make the default `'#---|##` or similar? The latter would be pretty much exactly like matlab, which might be a good default.